UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeSetUtilities.java

```diff
@@ -201,9 +202,11 @@ public boolean applyPropertyAlias(
                     }
                     ;
                     if (!status) {
-                        try {
-                            status = applyPropertyAlias0(prop, "No", result, !invert);
-                        } catch (Exception e) {
+                        if (prop.isType(UnicodeProperty.BINARY_OR_ENUMERATED_OR_CATALOG_MASK)) {
+                            try {
+                                status = applyPropertyAlias0(prop, "No", result, !invert);
```

Comment:
I understand that binary properties default to "No", but each enumerated/catalog property has its own `@missing` value, so why should those default to "No" rather than their actual `@missing` value?